sematic/api_client.py

```diff
@@ -104,8 +105,21 @@ def store_future_bytes(future_id: str, bytes_: bytes) -> None:
     _store_bytes("futures", future_id, bytes_)
 
 
+def store_file_content(file_path: str, namespace: str, key: str) -> None:
+    """
+    Store content of local file.
+    """
+    with open(file_path, "rb") as f:
+        bytes_ = f.read()
```

Comment:
This _could_ OOM the log streamer if the file being uploaded from is large. We only collect logs for 10s at a time, but a quick experiment on my laptop showed that it's possible to generate ~20GB of log lines in that time period. A more realistic "high volume logs" situation for these workloads might be more like one 500 character line every 1ms, which is ~4MB. So we're probably fine. A more proper implementation would stream from the file to the request.